app/views/amends/ViewReturnSummaryView.scala.html

```diff
@@ -17,24 +17,32 @@
 @import views.html.components.{SectionHeader, Link}
 @import models.NormalMode
 @import viewmodels.checkAnswers.ViewReturnSummaryViewModel
+@import config.{Features, FrontendAppConfig}
 
 @this(
     layout: templates.Layout,
     formHelper: FormWithCSRF,
     govukSummaryList: GovukSummaryList,
     govukButton: GovukButton,
     sectionHeader: SectionHeader,
-    link: Link
+    link: Link,
+    appConfig: FrontendAppConfig
 )
 
 @(returnPeriod: String, vm: ViewReturnSummaryViewModel)(implicit request: Request[_], messages: Messages)
 
+    @amendLink = @{if(appConfig.isAmendsFeatureEnabled) link("amend-return",
+                    messages("viewReturnSummary.amendLink"),
+                    controllers.amends.routes.AmendAreYouSureController.onPageLoad(NormalMode))
+                    }
+
 @layout(pageTitle = titleNoForm(returnPeriod), showBackLink = false) {
 
     @sectionHeader(messages("viewReturnSummary.sectionHeader"))
     <h1 class="govuk-heading-xl">@returnPeriod</h1>
 
-    <p>@link("amend-return", messages("viewReturnSummary.amendLink"), controllers.amends.routes.AmendAreYouSureController.onPageLoad(NormalMode))</p>
+    <p>@amendLink</p>
```

Comment:
this will put an empty p tag when switched off, is that correct, unsure how screen readers cope with that

Reply:
empty elements get skipped 👍

Reply:
Hmm, thought they were skipped, i'll check it